docs/core/compatibility/windows-forms/5.0/winforms-objects-not-accessible-from-native-code.md

```diff
@@ -0,0 +1,131 @@
+---
+title: "Breaking change: WinForms objects not accessible from native code"
+description: Learn about the breaking change in .NET 5.0 where Windows Forms objects are no longer accessible from native code.
+ms.date: 01/29/2021
+---
+# Native code can't access Windows Forms objects
+
+Starting in .NET 5.0, you can no longer access Windows Forms objects from native code.
+
+## Change description
+
+In previous .NET versions, some Windows Forms types were decorated as visible to COM interop, and thus were accessible to native code. Starting in .NET 5.0, no Windows Forms API are visible to COM interop or accessible to native code. The .NET runtime no longer supports creating custom type libraries out of the box. In addition, the .NET runtime can't depend on the type library for .NET Framework (which would require maintaining the shape of classes as they were in .NET Framework).
+
+## Reason for change
+
+- Removal of `ComVisible(true)` from enumerations that were used for TLB generation and lookup: Since there is no WinForms TLB provided by .NET Core, there is no value in keeping this attribute.
```

Comment:
Should the first mention of an acronym clarify what does it stand for between parentheses?

Reply:
I'd probably add the acronym to the above paragraph where "type libraries" is first mentioned.